index.html

```diff
@@ -5058,39 +5068,47 @@ <h3 id="playback-AudioBufferSourceNode">
             Otherwise, its return value is determined by a UA-supplied
             algorithm that interpolates between sample frames in the
             neighborhood of <code>position</code>. If <code>position</code> is
-            greater than or equal to <code>actualLoopEnd</code> and there is no
+            greater than or equal to <code>loopEnd</code> and there is no
             subsequent sample frame in <code>buffer</code>, then interpolation
             should be based on the sequence of subsequent frames beginning at
-            <code>actualLoopStart</code>.
-            </li>
-            <li>Let <code>bufferTime</code> equal 0. This is the <em>playhead
-            position</em> within <code>buffer</code> of the next output sample
-            frame. The initial value -1 indicates that this position has not
-            yet been determined.
+            <code>loopStart</code>.
             </li>
-            <li>Let <code>started</code> equal <code>false</code>, indicating
-            whether output of the buffer contents has begun or not.
-            </li>
-            <li>For each block of audio to be rendered:
+            <li>For each quantum of audio to be rendered:
```

Comment:
I don't think we need to have any kind of loop for a render quantum. I think the algorithm, should just run forever, without any consideration of a render quantum. This allows us the get rid of some loops and also `index`.  I think it's pretty clear that the algorithm must generate some output every sample frame.